build/ferretdb/eval-dev.Dockerfile

```diff
@@ -106,15 +106,23 @@ RUN --mount=type=cache,sharing=locked,target=/var/cache/apt <<EOF
 mkdir /tmp/cover /tmp/state
 chown postgres:postgres /tmp/cover /tmp/state
 
-apt install -y curl
+apt install -y curl supervisor
 curl -L https://pgp.mongodb.com/server-7.0.asc | apt-key add -
 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/debian bookworm/mongodb-org/7.0 main" | tee /etc/apt/sources.list.d/mongodb-org-7.0.list
 apt update
 apt install -y mongodb-mongosh
 EOF
 
 COPY --from=eval-dev-build /src/bin/ferretdb /usr/local/bin/ferretdb
-COPY build/ferretdb/99-start-ferretdb.sh /docker-entrypoint-initdb.d/
+
+# TODO https://github.com/FerretDB/FerretDB/issues/5043
+COPY --from=eval-dev-build /src/build/ferretdb/evaluation/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+COPY --from=eval-dev-build /src/build/ferretdb/evaluation/start-ferretdb.sh /start-ferretdb
+
+ENTRYPOINT ["/usr/bin/supervisord", "-c", "/etc/supervisor/conf.d/supervisord.conf"]
+
+# clear CMD set by postgresql base image
+CMD []
```

Comment:
https://docs.docker.com/reference/dockerfile/#understand-how-cmd-and-entrypoint-interact:

> If CMD is defined from the base image, setting ENTRYPOINT will reset CMD to an empty value.

Reply:
thanks :pray: